functions/poll.js

```diff
@@ -355,6 +361,15 @@ async function onIntakeAssigned(id, fields, meta) {
     return null;
   }
 
+  // If status is unchanged and lastSeenDeliveryVolunteer is not set
+  // assume this is a ticket that just hasn't had lastSeenDeliveryVolunteer backfilled
+  // and update the meta
+  if (fields.status === meta.lastSeenStatus && _.isEmpty(meta.lastSeenDeliveryVolunteer)) {
```

Comment:
I want to avoid spamming folks with DMs due to backfilling lastSeenDeliveryVolunteer into _meta